doc/architecture-decisions/0018-route-crd-and-contour-controller.md

```diff
@@ -0,0 +1,48 @@
+# 18. Add Contour as an Alternative Ingress Solution Provider
+
+Date: 2020-10-09
+
+## Status
+
+Proposal
+
+## Context
+In our efforts to allow more optionality in ingress solutions for CF-for-K8s, we
+want to allow the use of Contour as a potential alternative to Istio.
+
+### Proposed Design
+
+We propose:
+1. Adding a configuration option `ingress_solution_provider` to CF-for-K8s, with
+   potential values `istio` or `contour`.
+2. Creating another kubebuilder controller, similar to routecontroller, to
+   handle when `ingress_solution_provider: contour` is configured.
+3. Renaming `Route Controller` to `Istio Controller`
+
+![Proposed ContourControllerDesign](../assets/contour-controller.png)
```

Comment:
I think this image is missing from the PR

Reply:
:(

Reply:
oh no :(